src/chains/signals.py

```diff
@@ -1,72 +1,48 @@
 import logging
-from functools import cache
 from typing import Any
-from urllib.parse import urljoin
 
-import requests
 from django.conf import settings
 from django.db.models.signals import post_delete, post_save
 from django.dispatch import receiver
 
+import clients.safeclientgateway
+
 from .models import Chain, Feature, GasPrice, Wallet
 
 logger = logging.getLogger(__name__)
 
 
-@cache
-def setup_session() -> requests.Session:
-    session = requests.Session()
-    adapter = requests.adapters.HTTPAdapter(max_retries=3)
-    session.mount("http://", adapter)
-    session.mount("https://", adapter)
-    return session
-
-
-def _trigger_client_gateway_flush() -> None:
-    cgw_url = settings.CGW_URL
-    if cgw_url is None:
-        logger.error("CGW_URL is not set. Skipping hook call")
-        return
-    cgw_flush_token = settings.CGW_FLUSH_TOKEN
-    if cgw_flush_token is None:
-        logger.error("CGW_FLUSH_TOKEN is not set. Skipping hook call")
-        return
-
-    url = urljoin(cgw_url, "/v2/flush")
-    try:
-        post = setup_session().post(
-            url,
-            json={"invalidate": "Chains"},
-            headers={"Authorization": f"Basic {cgw_flush_token}"},
-        )
-        post.raise_for_status()
-    except Exception as error:
-        logger.error(error)
+def _flush_cgw_chains() -> None:
+    clients.safeclientgateway.flush(
```

Comment:
camel case?

Reply:
Shouldn't package/modules be all-lowercase? https://peps.python.org/pep-0008/#package-and-module-names

Reply:
Sorry. I was talking about snake case 😅

Reply:
Ah this is because it's not intended to be used outside of `signals.py`.

I looked at this paragraph:

> “Private” instance variables that cannot be accessed except from inside an object don’t exist in Python. However, there is a convention that is followed by most Python code: a name prefixed with an underscore (e.g. _spam) should be treated as a non-public part of the API (whether it is a function, a method or a data member). It should be considered an implementation detail and subject to change without notice.

https://docs.python.org/3.10/tutorial/classes.html#private-variables

Reply:
I'm taking about `safeclientgateway` 😅

Reply:
[dfda6b1](https://github.com/gnosis/safe-config-service/pull/469/commits/dfda6b1f9e3fb6f4d75390c81c92a17c9c3043cd)